src/util.ts

```diff
@@ -13,11 +13,32 @@ debugLog.log = console.log.bind(console);
 export const debugWarn = debug('electron-osx-sign:warn');
 debugWarn.log = console.warn.bind(console);
 
-const removePassword = function (input: string): string {
-  return input.replace(/(-P |pass:|\/p|-pass )([^ ]+)/, function (_, p1) {
-    return `${p1}***`;
-  });
-};
+function removePassword(input: string[]): string {
+  const secretFlags = new Set(['-P', '--password', '-pass', '/p', 'pass:']);
+
+  const redacted: string[] = [];
+  for (let i = 0; i < args.length; i++) {
```

Comment:
I don't think this compiles at the moment. Looks like we're using `args` as a variable instead of `input`.

Reply:
Yup, my bad, fixed.